### PR TITLE
Generalise statement beyond scientific stack

### PR DESCRIPTION
**Pull request by Carreau**
Just pick the commit @Zac-HD did on #95 about generalizing the statement and submit it as a PR. I think that was pretty non-controversial. 

We still likely want +1 from a few people.

### CODE REVIEW

File: _sections/20-statement.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ color: white
 
 # Moving to require Python 3
 
-All of the major projects making up the Scientific Python stack now support
+Almost all major open souce Python packages now support
```
Comment (mscuthbert) [inline _sections/20-statement.md]: souce to source

**Comment by asmeurer** [comment]
I agree with this change. I never liked that it said "scientific projects" in the first sentence.

File: _sections/20-statement.md (hunk 1 above)
Comment (Carreau) [reply _sections/20-statement.md]: Oh ! Thanks, fixed.

**Comment by takluyver** [comment]
Let's do this, thanks!

**Comment by Carreau** [comment]
Thanks !

**Pull request merged**